File: old_frienbly_web/contracts/src/main/java/com/platonic/contracts/user/replies/UserRegistrationReply.java
package com.platonic.contracts.user.replies;

import com.platonic.contracts.exceptions.ExceptionType;
import com.platonic.contracts.user.UserFieldType;

public class UserRegistrationReply {
    private ExceptionType exceptionType;
    private UserFieldType erroredField;

    public UserRegistrationReply() {}

    public UserRegistrationReply(ExceptionType exceptionType, UserFieldType fieldType) {
        this.exceptionType = exceptionType;
        this.erroredField = fieldType;
    }

    public ExceptionType getExceptionType() {
        return exceptionType;
    }

    public UserFieldType getErroredField() {
        return erroredField;
    }
}

File: old_frienbly_ngx/src/app/common/contracts/user/replies/user-registration.reply.ts
import {ExceptionType} from '../../exceptions/exception.type';

import {UserFieldType} from '../user-field.type';

export class UserRegistrationReply {
  private exceptionType: ExceptionType;
  private erroredField: UserFieldType;

  constructor(exceptionType: ExceptionType, erroredField: UserFieldType) {
    this.exceptionType = exceptionType;
    this.erroredField = erroredField;
  }

  getExceptionType(): ExceptionType {
    return this.exceptionType;
  }

  getErroredField(): UserFieldType {
    return this.erroredField;
  }
}
